pkg/utils/kubernetes/utils_test.go

```diff
@@ -0,0 +1,18 @@
+package kubernetes
+
+import (
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"testing"
+)
+
+func TestCustomResourceDefinitionExists(t *testing.T) {
+	gvk :=  schema.GroupVersionKind{Group: "console.openshift.io", Version: "v1", Kind: "consoleyamlsamples"}
+	res, err := CustomResourceDefinitionExists(gvk, nil)
+	assert.Error(t, err)
+	assert.Equal(t, false, res, "Expected should equal to actual result")
+
+	res, err = CustomResourceDefinitionExists(gvk)
+	assert.NoError(t, err)
+	assert.Equal(t, true, res, "Expected should equal to actual result")
+}
```

Comment:
It is usually good form to keep the newline at eof.

Reply:
added